components/crypto_dot_com/browser/crypto_dot_com_pref_utils.h

```diff
@@ -0,0 +1,23 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_COMPONENTS_CRYPTO_DOT_COM_BROWSER_CRYPTO_DOT_COM_PREF_UTILS_H_
+#define BRAVE_COMPONENTS_CRYPTO_DOT_COM_BROWSER_CRYPTO_DOT_COM_PREF_UTILS_H_
+
+class PrefRegistrySimple;
+
+namespace crypto_dot_com {
+
+class CryptoDotComPrefUtils {
+ public:
+  CryptoDotComPrefUtils();
+  ~CryptoDotComPrefUtils();
```

Comment:
Seems like there's no need for a constructor/destructor or even to have a class here at all. You can just make it a function.

Reply:
just doing a plain function ow